conn/spi/spi.go

```diff
@@ -114,23 +128,15 @@ type Conn interface {
 	//
 	// mode specifies the clock and signal polarities, if the bus is using half
 	// duplex (shared MISO and MOSI) or if CS is not needed.
-	DevParams(maxHz int64, mode Mode, bits int) error
-	// TxPackets does multiple operations over the SPI bus.
-	//
-	// The maximum number of bytes can be limited depending on the driver. Query
-	// conn.Limits.MaxTxSize() can be used to determine the limit.
-	//
-	// If the last packet has KeepCS:true, the behavior is undefined. The CS line
-	// will likely not stay asserted. This is a driver limitation.
-	TxPackets(p []Packet) error
+	DevParams(maxHz int64, mode Mode, bits int) (Conn, error)
 }
```

Comment:
Have you considered calling `ConnConfig` a `Bus`, and calling `DevParams` either `Connect` or `Device`? You are opening the bus and you connect to a specific device on the bus by proving the params. Mhh, The CnnConfig already has the chip-select built-in, why? I'm probably missing something :-).